_posts/2017/2017-02-23-scrum-interview.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Scrum Master insights at ePages"
+date: "2017-02-09 07:00:00"
+categories: working-at-epages
+authors: ["Rose", "Ann-Katrin"]
+---
+
+As already argued in our previous [blog post](https://developer.epages.com/blog/2017/01/31/getting-twice-the-things-done-in-half-the-time.html) the agile process framework [Scrum](https://www.scrum.org/resources/what-is-scrum) is our preferred methodology in order to manage our product development at ePages.
+To get further insights into our work with Scrum, we interviewed our Scrum Master Ann-Katrin:
+
+### Ann-Katrin, please introduce yourself!
```

Comment:
Maybe you could also go with: "Hi Ann-Katrin. Many thanks for taking the time to do this interview. Please introduce yourself.